src/xngin/apiserver/sqla/tables.py

```diff
@@ -490,13 +485,7 @@ class Arm(Base):
     """Representation of arms of an experiment."""
 
     __tablename__ = "arms"
-    # TODO: Ensure arm names are unique within an organization
-    #       Do this as part of Issue #278; will need to backfill in such a way that old arms are
-    #       made unique e.g. suffixing with a part of the experiment id.
-    # __table_args__ = (
-    #     sqlalchemy.UniqueConstraint("name", "organization_id", name="uix_arm_name_org"),
-    # )
-    # -- Arm metadata --
+
```

Comment:
note to self: fwiw, whenever we get around to this, probably best done as a new column with a unique constraint nowadays